### ISSUE TITLE
[Campaign Close] Dynamic Reportback gallery

### ISSUE THREAD

**Issue by aaronschachter** [DoSomethingArchive/legacy-website#2394]
Depends on #2356.  If there is a flag in place to promote Reportback Files, we can display the top X promoted instead of implementing this via a Reportback File reference field #2393.

For short term, we'll implement #2393 first as it's a much smaller lift, so I'm labelling this issue as low priority for now.

This would also apply a Campaign Action Page, could use the smae functionality instead of the Image node Entityreference field for the "Prove It" gallery.

**Comment by aaronschachter** [comment]
Ready for theming. Assigning over to @tjrosario
